### DISCUSSION TITLE
Make timestamp parameter `t` optional

### DISCUSSION THREAD

**Discussion by PaxxPatriot** [miguelgrinberg/python-engineio#385] [Ideas]
According to the JavaScript socket.io documentation, there is an option to deactivate the addition of the cache-busting timestamp parameter: https://socket.io/docs/v4/client-options/#timestamprequests

This is also reflected in the JavaScript code here: https://github.com/socketio/socket.io/blob/main/packages/engine.io-client/lib/transports/websocket.ts#L142. It is also disabled by default.

The behaviour of the current python-engineio implementation was blocking me from using socket.io with the Skinport socket.io Server.
So my suggestion is to make it possible to disable the addition of this parameter to the initial request URL.

**Comment by miguelgrinberg** [comment]
Added in https://github.com/miguelgrinberg/python-engineio/commit/e8f9bbafc8c3ef8126dbe06343d0a30e32074627

**Comment by PaxxPatriot** [comment]
Thanks, I tested this already and it works perfectly fine now. Thanks for the quick implementation!
